msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -1063,7 +1063,7 @@ The `extensions.ribbons` property provides the ability to add [add-in commands](
 |`tabs.customMobileRibbonGroups.controls.id` | String | 250 characters | ✔️ | Specifies the ID of the control such as "msgReadFunctionButton".|
 |`tabs.customMobileRibbonGroups.controls.type` | String enum |  | ✔️ | Specifies the type of control. Currently only "MobileButton" is supported.|
 |`tabs.customMobileRibbonGroups.controls.label` | String | 32 characters | ✔️ | Specifies the label on the control.|
-|`tabs.customMobileRibbonGroups.controls.actionId` | String | 64 characters |✔️ | Specifies the ID of the action that is taken when a user selects the control. The `actionId` must match the `runtime.actions.id` property of an action in the `runtimes` object.|
+|`tabs.customMobileRibbonGroups.controls.actionId` | String | 64 characters | | Specifies the ID of the action that is taken when a user selects the control. The `actionId` must match the `runtime.actions.id` property of an action in the `runtimes` object.|
```

Comment:
The property `actionId` in the  `extensionCommonCustomControlMenuItem` is marked as required in the [Dev Preview schema](https://developer.microsoft.com/json-schemas/teams/vDevPreview/MicrosoftTeams.schema.json).

Is the update coming as part of a new schema version? Request you to please check and confirm.

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/93579821/3233253e-1746-4278-83e5-356b6acbfaa8)

Reply:
Confirmed with final schema update